### ISSUE TITLE
Autofill suggestion preview

### ISSUE THREAD

**Issue by AdiWeit** [typst/webapp-issues#184]
### Description

I am not sure how doable this would be outside of the web app, so I am suggesting it here:
While typing, the web app gives you auto-fill suggestions. It would be great if it would be able to show how it would look like with the selected auto-fill suggestion without the need to use the suggestion in the code. I imagine it being similar to the HTML/Javascript debug window like seen in the video attached below:

https://github.com/typst/webapp-issues/assets/43521682/b3eb8ef8-a839-41eb-a591-3dcff6a21796

### Use Case

I think this would be useful if you don't know which of the suggested options is the one you want because having this you won't need to look it up in the documentation or try writing all options and deleting them again. 
The use case because of which I came up with this idea was that I wanted the `$angle.spheric$`-symbol, but I only knew it has to be something with `angle.`. I ended up searching in the documentation.

**Comment by laurmaedje** [comment]
I fully agree with you that this should be a thing. We definitely want to implement this, but haven't gotten to it yet. It also needs some care because, for example, with fonts it might be problematic if we eagerly download all the fonts you skip over while typing in `set text(font: ..)`.

**Comment by MauritsWilke** [comment]
> I fully agree with you that this should be a thing. We definitely want to implement this, but haven't gotten to it yet. It also needs some care because, for example, with fonts it might be problematic if we eagerly download all the fonts you skip over while typing in `set text(font: ..)`.

For the case of font selecting, perhaps showing the font in the selection menu (instead of rendering the entire document again) might be a solution. Showing the names of the fonts in their own font like Google Docs and Microsoft Word currently do would be a good example.
